gcp/terraform/output.tf

```diff
@@ -1,4 +1,5 @@
 output "END_USER_SETUP_INSTRUCTIONS" {
+  depends_on = [module.services]
```

Comment:
This ensures that the instructions are actually outputted when we `terraform apply -target="module.services"`.  Was already fixed in this way in Azure.